pynest/examples/evaluate_quantal_stp_synapse.py

```diff
@@ -69,124 +69,156 @@
        http://dx.doi.org/10.1016/S0893-6080(01)00144-7
 
 """
+
 import nest
-import nest.voltage_trace
 import numpy
 import matplotlib.pyplot as plt
 
-nest.ResetKernel()
-
 ################################################################################
 # On average, the ``quantal_stp_synapse`` converges to the ``tsodyks2_synapse``,
 # so we can compare the two by running multiple trials.
 #
-# First we define the number of trials as well as the number of release sites.
+# First we define simulation time step and random seed
+
+resolution = 0.1  # [ms]
+seed = 12345
+
+# We define the number of trials as well as the number of release sites.
 
 n_syn = 10.0  # number of synapses in a connection
 n_trials = 100  # number of measurement trials
```

Comment:
In the comment for `n_syn` I'd write: number of release sites.

For `n_trials` maybe it's better to change its value to 1000, as I mentioned in the other comment, with this I get a smaller deviation between the deterministic and the probabilistic models.

Reply:
I like that ;)